test/support/generator/post-emails-generator.ts

```diff
@@ -0,0 +1,95 @@
+import { GOVUK_NOTIFY } from '@ukef/constants';
+import { PostEmailsResponseDto } from '@ukef/helper-modules/govuk-notify/dto/post-emails-response.dto';
+import { PostEmailsRequestItemDto } from '@ukef/modules/emails/dto/post-emails-request.dto';
+
+import { AbstractGenerator } from './abstract-generator';
+import { RandomValueGenerator } from './random-value-generator';
+
+export class PostEmailsGenerator extends AbstractGenerator<PostEmailsValues, GenerateResult, GenerateOptions> {
+  constructor(protected readonly valueGenerator: RandomValueGenerator) {
+    super(valueGenerator);
+  }
+
+  protected generateValues(): PostEmailsValues {
+    const templateId = this.valueGenerator.string({ length: GOVUK_NOTIFY.FIELD_LENGTHS.TEMPLATE_ID });
+    return {
+      templateId,
+      reference: `${templateId}-12345`,
+      sendTransactionId: this.valueGenerator.string({ length: GOVUK_NOTIFY.FIELD_LENGTHS.TRANSACTION_ID }),
+      serviceId: this.valueGenerator.string({ length: GOVUK_NOTIFY.FIELD_LENGTHS.SERVICE_ID }),
+      fromEmail: this.valueGenerator.email(),
+      toEmail: this.valueGenerator.email(),
+      emailSubject: this.valueGenerator.sentence(),
+      emailBody: this.valueGenerator.paragraph({ sentences: 3 }),
+      personalisation: {
+        firstName: this.valueGenerator.word(),
+        lastName: this.valueGenerator.word(),
+      },
+    };
+  }
+
+  protected transformRawValuesToGeneratedValues(values: PostEmailsValues[], _options: GenerateOptions): GenerateResult {
+    const requests: PostEmailsRequestItemDto[][] = values.map((v) => [
+      {
+        templateId: v.templateId,
+        sendToEmailAddress: v.toEmail,
+        personalisation: v.personalisation,
+      },
+    ]);
+
+    const mdmPath = '/api/v1/emails';
```

Comment:
Constant?

Reply:
I did search and this is the only place where this `/emails` is used. And we don't usualy have constants for `/api/v1` and domain path.